source/type/stop_point.h

```diff
@@ -53,6 +53,7 @@ struct StopPoint : public Header, Nameable, hasProperties, HasMessages {
 
     StopArea* stop_area;
     std::vector<navitia::georef::Admin*> admin_list;
+    std::set<navitia::type::AccessPoint*> access_points;
```

Comment:
An AccessPoint has to be unique for the StopPoint point of view.
This is the reason of the set